app/routes/meetings/edit.js

```diff
@@ -6,8 +6,38 @@ export default class MeetingsEditRoute extends Route {
 
   async model(params) {
     const zitting = await this.store.findRecord('zitting', params.id, {
-      include: 'bestuursorgaan,secretaris,voorzitter,intermissions',
+      include:
+        'bestuursorgaan,secretaris,voorzitter,intermissions,publicatie-agendas',
     });
-    return zitting;
+    const publicationFilter = {
```

Comment:
it might make more sense to load this in the delete meeting component and have this async, though I could argue either way.

Reply:
So that the delete meeting component decides whether it is shown or not?